### PR TITLE
fix select2 ajax filter clearing

### PR DESCRIPTION
**Pull request by alashow**
It wasn't clearing selected choice when Remove all filters clicked, because of the wrong selector.

To reproduce:
- Search and select some option
- Click "Remove all filters"
- The filter is not activated, but the last choice is still selected. Search and select the same choice, the filter will not be activated. Because the choice is the same.

`$("li[filter-name={{ $filter->name }}] .select2")` doesn't select anything. Changed selector to same selector as [initializing selector](https://github.com/alashow/CRUD/blob/dev/src/resources/views/filters/select2_ajax.blade.php#L54).

### CODE REVIEW

File: src/resources/views/filters/select2_ajax.blade.php [hunk 1]
```diff
@@ -140,7 +140,7 @@ class="dropdown {{ Request::get($filter->name)?'active':'' }}">
 			$("li[filter-name={{ $filter->name }}]").on('filter:clear', function(e) {
 				// console.log('select2 filter cleared');
 				$("li[filter-name={{ $filter->name }}]").removeClass('active');
-				$("li[filter-name={{ $filter->name }}] .select2").select2("val", "");
+                $('#filter_{{ $filter->name }}').select2("val", "");
```
Comment (lloy0076) [inline src/resources/views/filters/select2_ajax.blade.php]: Can we change all the quotes in this line to either single or double quotes (but not a random mixture of both)?

Comment (alashow) [inline src/resources/views/filters/select2_ajax.blade.php]: Hope I didn't overkill it with 5c17f0a.

Comment (alashow) [inline src/resources/views/filters/select2_ajax.blade.php]: Hope I didn't overkill it in a5d10e3.

Comment (tabacitu) [reply src/resources/views/filters/select2_ajax.blade.php]: Haha. No, it's perfect, thank you :-) I wish we had this kind of consistency across all Backpack JS, but hey, we'll get there soon enough :-)

**Comment by tabacitu** [comment]
Thank you @alashow . Just tested it and merged it into the dev branch. It will be available to everyone this week. Cheers and thank you for using and improving Backpack!

**Pull request merged**